## QA Track 2 Review — FAIL

**Test suite: 1 failure**

`tests/discovery/manifests.test.ts > tool manifest generation > matches the committed generated JSON artifact` fails because `docs/generated/tool-manifest.json` was not regenerated after the description changes.

### Failure output (verbatim)

```
FAIL  tests/discovery/manifests.test.ts > tool manifest generation > matches the committed generated JSON artifact
AssertionError: expected { schemaVersion: '1.0', …(3) } to deeply equal { schemaVersion: '1.0', …(3) }

- Expected (generated from source)
+ Received (committed JSON on disk)

-       "description": "Delete an assignment from a course. This action is permanent.",
+       "description": "Delete an assignment from a course.",
...same pattern for delete_discussion, delete_page, delete_peer_review, remove_enrollment
```

### Fix required

Run `pnpm generate:manifests` and commit the updated `docs/generated/tool-manifest.json` to this branch.

### What passed

- `pnpm typecheck` ✅ — zero errors
- `pnpm lint` ✅ — ESLint + Prettier clean
- Spec alignment ✅ — all 5 target tools updated; destructiveHint sweep confirmed no missed tools
- Code quality ✅ — description-only changes, handlers/schemas/annotations untouched

55 test files, 601 tests pass. Only the manifest artifact check fails.

--- a/docs/generated/tool-manifest.json
+++ b/docs/generated/tool-manifest.json
@@ -144,7 +144,7 @@
     {
       "name": "delete_assignment",
       "domain": "assignments",
-      "description": "Delete an assignment from a course.",
+      "description": "Delete an assignment from a course. This action is permanent.",
       "annotations": {
         "destructiveHint": true,
         "idempotentHint": true,
@@ -582,7 +582,7 @@
     {
       "name": "remove_enrollment",
       "domain": "enrollments",
-      "description": "Remove or conclude an enrollment from a course.",
+      "description": "Remove or conclude an enrollment from a course. The 'delete' task is permanent; 'conclude' and 'deactivate' are reversible.",
       "annotations": {
         "destructiveHint": true,
         "openWorldHint": true
@@ -666,7 +666,7 @@
     {
       "name": "delete_discussion",
       "domain": "discussions",
-      "description": "Delete a discussion topic from a course.",
+      "description": "Delete a discussion topic from a course. This action is permanent.",
       "annotations": {
         "destructiveHint": true,
         "openWorldHint": true
@@ -798,7 +798,7 @@
     {
       "name": "delete_page",
       "domain": "pages",
-      "description": "Delete a wiki page from a course.",
+      "description": "Delete a wiki page from a course. This action is permanent.",
       "annotations": {
         "destructiveHint": true,
         "openWorldHint": true
@@ -930,7 +930,7 @@
     {
       "name": "delete_peer_review",
       "domain": "peer_reviews",
-      "description": "Remove a peer review assignment from a submission.",
+      "description": "Remove a peer review assignment from a submission. This action is permanent.",
       "annotations": {
         "destructiveHint": true,
         "openWorldHint": true

